charts/dependentapi-simple-operator/templates/serviceinventoryapi-service.yaml

```diff
@@ -0,0 +1,17 @@
+{{ if .Values.serviceInventoryAPI.enabled -}}
+apiVersion: v1
+kind: Service
+metadata:
+  name: info  # do not use {{ .Release.Name }}-info-service, 
+              # because the service has to be accessible under the 
+              # fixed endpoint info.canvas.svc.cluster.local
+  labels:
+    app: {{ .Release.Name }}-info-service
+spec:
+  ports:
+  - name: http
+    port: 80
+    targetPort: 8638
```

Comment:
we use http here, not https?

Reply:
yes, the tls termination is done by the ingress gateway. At the moment all our services are http, even our HashiCorp Vault uses http. There is a feature story to secure communication inside the cluster, but not yet started.